src/egon/data/datasets/power_plants/wind_offshore.py

```diff
@@ -0,0 +1,186 @@
+import pandas as pd
+import geopandas as gpd
+import numpy as np
+from egon.data import db
+from shapely.geometry import Point
+from pathlib import Path
+import egon.data.config
+
+
+def insert():
+    """
+    Include the offshore wind parks in egon-data
+    locations and installed capacities based on: NEP2035_V2021_scnC2035
+
+    Parameters
+    ----------
+    *No parameters required    
+    """
+    
+    cfg = egon.data.config.datasets()["power_plants"]
+    offshore_path = (
+        Path(".")
+        / "data_bundle_egon_data"
+        / "nep2035_version2021"
+        / cfg["sources"]["nep_2035"]
+    )
+
+    offshore = pd.read_excel(
+        offshore_path,
+        sheet_name="WInd_Offshore_NEP",
+        usecols=["Netzverknuepfungspunkt", "Spannungsebene in kV", "C 2035"],
+    )
+    offshore.dropna(subset=["Netzverknuepfungspunkt"], inplace=True)
+
+    # Import manually generated list of wind offshore farms with their connexion
+    # points (OSM_id)
+    id_bus = {
+        "Büttel": "w136034396",
+        "Heide/West": "w30622610",
```

Comment:
I'm a bit wondering why this substation is so far away from Heide. Why did you choose this osm_id?

Reply:
It is because of the required voltage level. The closest substations to Heide are "UW Husum Nord" and "Umspannwerk Kernkraftwerk Brunsbüttel" (this one was selected). They are probably the same distance from Heide. If you consider it necessary, we can agree on a closer point with a different voltage level.

Reply:
The connection point was changed to a closer bus.